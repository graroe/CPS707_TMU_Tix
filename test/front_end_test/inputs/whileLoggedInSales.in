login
sales
create
add
delete
logout