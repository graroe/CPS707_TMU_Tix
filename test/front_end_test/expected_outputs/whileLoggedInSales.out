Enter session type <sales or admin>: 
Sales agent login successful.
Command not authorized for sales account.
Command not authorized for sales account.
Command not authorized for sales account.
Session terminated.